examples/api/lib/widgets/animated_switcher/animated_switcher.0.dart

```diff
@@ -11,19 +11,27 @@ void main() => runApp(const AnimatedSwitcherExampleApp());
 class AnimatedSwitcherExampleApp extends StatelessWidget {
   const AnimatedSwitcherExampleApp({super.key});
 
+  static const Duration duration = Duration(microseconds: 500);
```

Comment:
Is there a reason we need static duration?

Reply:
@TahaTesser thanks for the review!
I made the duration static const to reference it in the test.

Reply:
I don't think we should add unnecessary static variables just for the tests. 

Please revert and hard code duration in the test.